website/docs/r/sentinel_alert_rule_ms_security_incident.html.markdown

```diff
@@ -56,12 +56,18 @@ The following arguments are supported:
 
 ---
 
+* `alert_rule_template_name` - (Optional) The name of the alert rule template which is used to create this Sentinel Scheduled Alert Rule. Changing this forces a new Sentinel MS Security Incident Alert Rule to be created.
```

Comment:
If this can only be a guid, wouldn't';t this make more sense as
```suggestion
* `alert_rule_template_guid` - (Optional) The name of the alert rule template which is used to create this Sentinel Scheduled Alert Rule. Changing this forces a new Sentinel MS Security Incident Alert Rule to be created.
```

Reply:
updated